modules/caddyhttp/fileserver/browse.go

```diff
@@ -106,6 +106,7 @@ func (fsrv *FileServer) browseApplyQueryParams(w http.ResponseWriter, r *http.Re
 	sortParam := r.URL.Query().Get("sort")
 	orderParam := r.URL.Query().Get("order")
 	limitParam := r.URL.Query().Get("limit")
+	fromParam := r.URL.Query().Get("from")
```

Comment:
As discussed, change to `offset`.
```suggestion
	offsetParam := r.URL.Query().Get("offset")
```